docs/WALLET_SDK.md

```diff
@@ -0,0 +1,125 @@
+# Fuel Wallet SDK
+
+- [Fuel Wallet SDK](#fuel-wallet-sdk)
+  - [How to use?](#how-to-use)
+  - [Quickstart](#quickstart)
+    - [Request connection](#request-connection)
+    - [List user accounts](#list-user-accounts)
+    - [Signing a message](#signing-a-message)
+  - [SDK API](#sdk-api)
+    - [Methods](#methods)
+      - [Connect](#connect)
+      - [Disconnect](#disconnect)
+      - [List Accounts](#list-accounts)
+      - [Request signature message](#request-signature-message)
+    - [Events](#events)
+      - [Accounts](#accounts)
+      - [Connection](#connection)
+
+## How to use?
+
+When a user has the FuelWallet extension already installed on the browser you will be able to,
+interact with the Wallet API by access our SDK on `window.FuelWeb3`.
+
+## Quickstart
+
+### Request connection
+
+Before any action you need to request the user to connect and authorized your application, via the wallet.
```

Comment:
First of all, you need to connect and authorized your application to use the user's FuelWallet extension.